src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.state.ColorSupplier;
+import core.basesyntax.state.Figure;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String name) {
+        this.name = name;
+        color = ColorSupplier.getRandomColor();
+        radius = (int)((Math.random() * 10) + 1);
```

Comment:
Add this variable to constructor parameters. Same in all places in you're code

Reply:
do u mean color?

Reply:
done, i guess